schema/rdl_schema_0.1.json

```diff
@@ -462,7 +421,7 @@
                   "description": "The methodology used for the calculation of the function"
                 },
                 "approach": {
-                  "$ref": "#/$defs/vulnerability_f_subtype",
```

Comment:
Like in https://github.com/GFDRR/rdl-standard/pull/101#issuecomment-1613842371 and https://github.com/GFDRR/rdl-standard/pull/105#issuecomment-1613825081, looks like this fix from the `dev` branch has been overwritten when resolving merge conflicts.

Reply:
Fixed in https://github.com/GFDRR/rdl-standard/pull/102/commits/e62a4b68f2544f468578d32f0848e0349a1bc9e2